data-es2016plus.js

```diff
@@ -6993,6 +6993,118 @@ exports.tests = [
       rhino1_7_15: true
     }
   },
+  {
+    name: 'Change Array by copy',
+    category: '2023 features',
+    significance: 'small',
+    spec: 'https://github.com/tc39/proposal-change-array-by-copy',
+    subtests: [
+      {
+        name: "Array.prototype.toReversed",
+        exec: function () {/*
+          var arr = [1,2,3];
+          return arr.toReversed()[0] === 3;
```

Comment:
```suggestion
          var arr = [1, 2, 3];
          return arr.toReversed()[0] === 3 && arr[0] === 1;
```
to ensure the original isn't mutated